src/sign.c

```diff
@@ -1557,329 +1532,328 @@ sign_jump_cmd(
+    buf_T *buf = NULL;
 
     // Parse the subcommand.
     p = skiptowhite(arg);
     idx = sign_cmd_idx(arg, p);
     if (idx == SIGNCMD_LAST)
     {
-	semsg(_(e_unknown_sign_command_str), arg);
-	return;
+        semsg(_(e_unknown_sign_command_str), arg);
+        return;
     }
     arg = skipwhite(p);
 
     if (idx <= SIGNCMD_LIST)
     {
-	// Define, undefine or list signs.
-	if (idx == SIGNCMD_LIST && *arg == NUL)
-	{
-	    // ":sign list": list all defined signs
-	    for (sp = first_sign; sp != NULL && !got_int; sp = sp->sn_next)
-		sign_list_defined(sp);
-	}
-	else if (*arg == NUL)
-	    emsg(_(e_missing_sign_name));
-	else
-	{
-	    char_u	*name;
-
-	    // Isolate the sign name.  If it's a number skip leading zeroes,
-	    // so that "099" and "99" are the same sign.  But keep "0".
-	    p = skiptowhite(arg);
-	    if (*p != NUL)
-		*p++ = NUL;
-	    while (arg[0] == '0' && arg[1] != NUL)
-		++arg;
-	    name = vim_strsave(arg);
-
-	    if (idx == SIGNCMD_DEFINE)
-		sign_define_cmd(name, p);
-	    else if (idx == SIGNCMD_LIST)
-		// ":sign list {name}"
-		sign_list_by_name(name);
-	    else
-		// ":sign undefine {name}"
-		sign_undefine_by_name(name, TRUE);
-
-	    vim_free(name);
-	    return;
-	}
+        // Define, undefine or list signs.
+        if (idx == SIGNCMD_LIST && *arg == NUL)
+        {
+            // ":sign list": list all defined signs
+            for (sp = first_sign; sp != NULL && !got_int; sp = sp->sn_next)
+                sign_list_defined(sp);
+        }
+        else if (*arg == NUL)
+            emsg(_(e_missing_sign_name));
+        else
+        {
+            char_u *name;
+
+            // Isolate the sign name.  If it's a number skip leading zeroes,
+            // so that "099" and "99" are the same sign.  But keep "0".
+            p = skiptowhite(arg);
+            if (*p != NUL)
+                *p++ = NUL;
+            while (arg[0] == '0' && arg[1] != NUL)
+                ++arg;
+            name = vim_strsave(arg);
+
+            if (idx == SIGNCMD_DEFINE)
+                sign_define_cmd(name, p);
+            else if (idx == SIGNCMD_LIST)
+                // ":sign list {name}"
+                sign_list_by_name(name);
+            else
+                // ":sign undefine {name}"
+                sign_undefine_by_name(name, TRUE);
+
+            vim_free(name);
+            return;
+        }
     }
     else
     {
-	int		id = -1;
-	linenr_T	lnum = -1;
-	char_u		*sign_name = NULL;
-	char_u		*group = NULL;
-	int		prio = -1;
+        int id = -1;
+        linenr_T lnum = -1;
+        char_u *sign_name = NULL;
+        char_u *group = NULL;
+        int prio = -1;
 
-	// Parse command line arguments
-	if (parse_sign_cmd_args(idx, arg, &sign_name, &id, &group, &prio,
-							  &buf, &lnum) == FAIL)
-	    return;
+        // Parse command line arguments
+        if (parse_sign_cmd_args(idx, arg, &sign_name, &id, &group, &prio, &buf,
+                                &lnum)
+            == FAIL)
```

Comment:
That `== FAIL` on a separate line looks un-usual

Reply:
This and the other below should be resolved